pages/testComponent.tsx

```diff
@@ -1,7 +1,16 @@
+'use client'
+import {useState, useEffect} from 'react';
+
 const TestComponent = () => {
+    const [myState, setMyState] = useState(0);
+    useEffect(()=>{
+        setMyState(8);
+    },[]);
```

Comment:
```suggestion
    }, []);
```
That's just formatting, it's up to you if you want to reformat it.